packages/fulfilment-job/src/__mocks__/aws-sdk.js

```diff
@@ -1,3 +1,3 @@
 import { configureAwsSdkMock } from '../../../connectors-lib/src/__mocks__/aws-mock-helper.js'
-const AwsSdk = configureAwsSdkMock(jest.genMockFromModule('aws-sdk'))
+const AwsSdk = configureAwsSdkMock()
```

Comment:
Could we change the import to '@defra-fish/connectors-lib/src/__mocks__/aws-mock-helper.js' rather than what it is?